.gitlab-ci.yml

```diff
@@ -364,6 +364,27 @@ fedora upgrade:
       - new-installed-database.sql
       - new-upgraded-database.sql
 
+flawfinder: 
+  stage: test
+  variables:
+    GIT_STRATEGY: fetch
+    GIT_SUBMODULE_STRATEGY: normal
+  script:
+    - yum install -y python3 python3-pip git findutils file
+    - pip install flawfinder
+    - flawfinder --falsepositive --quiet --html . > flawfinder-all-vulnerabilties.html
+    - flawfinder --falsepositive --quiet --minlevel=5 --diffhitlist=flawfinder-ignorelist . | tee flawfinder.log
+    - PASS_STATUS=$(grep "No hits found." flawfinder.log)
+    - |
+      if [ "${PASS_STATUS}" == "No hits found." ]; then
```

Comment:
Nit: these 3 lines could just be `if grep -q "No hits found." flawfinder.log; then`